content/docs/command-reference/params/index.md

```diff
@@ -43,7 +43,9 @@ to define [stage](/doc/command-reference/run) dependencies more granularly:
 changes to other parts of the params file will not affect the stage. Parameter
 dependencies also prevent situations where several stages share a regular
 dependency (e.g. a config file), and any change in it invalidates all of them
-(see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
+(see `dvc status`), causing unnecessary re-executions upon `dvc repro`. However,
+you can track all the parameters in a file by setting _parameter value_ to be
+`null`.
```

Comment:
I am not quite sure on how to include this information here.

Reply:
I think an example either here or in the examples section below would clarify better than any explanation.

Reply:
It would help to show both `dvc.yaml` and how `dvc stage add` can be used.

Some rationale for it might also help, like explaining that you may want to granularly track all parameters in a file and it would be tedious to list them all and keep them updated as new parameters are added.

Reply:
No need to include this info. in this paragraph I think. This is a general explanation about parameter dependencies.

How about adding the note or even a new paragraph and sample block to https://dvc.org/doc/user-guide/project-structure/pipelines-files#parameter-dependencies instead?

Agree about updating `stage add` (as I mentioned in another comment) and on mentioning motivation when presenting this function

Reply:
Will update the other refs as agreed. Deleting this for now...
```suggestion
(see `dvc status`), causing unnecessary re-executions upon `dvc repro`.
```